workers.tf

```diff
@@ -12,6 +12,7 @@ resource "aws_autoscaling_group" "workers" {
       ]
     )
   )
+  wait_for_capacity_timeout = var.worker_asg_create_timeout
```

Comment:
This should be defined per ASG. You can add `asg_wait_for_capacity_timeout` default in [`local.workers_group_defaults_defaults`](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/local.tf#L26-L31).

Reply:
Maybe having this setting per worker group is too much? Can you imagine wanting a timeout of say 10m for one worker group but 20m for another?

Reply:
Not really in my case. But it's more to be consistent with other ASG options.

And by putting into the workers_group_defaults_defaults, you can achieve both defaults for all ASG and custom timeout per ASG (for those people with weird use cases :)) for free.

Reply:
OK cool, sounds good 🙂